File: packages/java/src/main/java/com/logolabs/inkvec/InternalException.java
package com.logolabs.inkvec;

/** The tracer failed or panicked. Not the caller's fault; worth a bug report. */
public final class InternalException extends InkvecException {
    private static final long serialVersionUID = 1L;

    InternalException(String message) {
        super(message);
    }
}

File: packages/php/src/InternalException.php
<?php

declare(strict_types=1);

namespace LogoLabs\Inkvec;

/**
 * The tracer itself failed or panicked (`INKVEC_ERR_INTERNAL`). Not the caller's fault:
 * worth a bug report at https://github.com/logolabs/inkvec/issues.
 */
final class InternalException extends InkvecException
{
}
